shared/test/ui_test_credentials.json

```diff
@@ -1,5 +1,13 @@
-{
-    "username" : "__INSERT__USERNAME__HERE__",
-    "password" : "__INSERT__PASSWORD__HERE__",
-    "url" : "test.salesforce.com"
-}
\ No newline at end of file
+[
```

Comment:
Hi, @khawkins , to my understanding, this is a standard JSON structure, it's an array of dictionary objects.. Or you want to wrap this array into an object?